src/univariate/continuous/beta.jl

```diff
@@ -39,8 +37,22 @@ end
 Beta(α::T, β::T) where {T<:Real} = Beta{T}(α, β)
```

Comment:
```suggestion
Beta(α::T, β::T) = Beta{T}(α, β)
```

Reply:
and
https://github.com/JuliaStats/Distributions.jl/pull/823/files#diff-8c151605b43426a03834a31681aab1d1R27

Reply:
should be `Beta(α::T, β::T) where {T} = Beta{T}(α, β)` ?